pkg/flink/types/processed_statement.go

```diff
@@ -94,10 +104,10 @@ func (s ProcessedStatement) GetPageSize() int {
 
 func (s ProcessedStatement) PrintStatementDoneStatus() {
 	if s.Status != "" {
-		output.Printf(false, "Statement phase is %s.\n", s.Status)
+		output.Printf(false, "Finished statement execution. Statement phase: %s.\n", s.Status)
 	}
 	if s.StatusDetail != "" {
-		output.Printf(false, "%s.\n", s.StatusDetail)
+		output.Printf(false, "Details: %s.\n", s.StatusDetail)
```

Comment:
Are the "details" here always warnings? It seems a bit awkward to do `Details: [Warning]`

Reply:
No, we can't assume it's a warning. This is a generic "Details" field where the backend can provide extra information about the statement.